tests/full_matrix/full_matrix_09.cc

```diff
@@ -41,7 +41,8 @@ void test()
 
   C.add(-1., E);
 
-  deallog << "Difference 1: " << (double)C.l1_norm() << std::endl;
+  const double tolerance = 100*std::numeric_limits<Number>::epsilon();
```

Comment:
should not be `const Number`?

Reply:
I did set the type of `tolerance` to `double` in `tests.h` to (possibly) support complex numbers, and I don't think it really matters as long as we have real types...

Reply:
sure, I know it's nitpicking, but you will have some issues in this line also for complex numbers as `std::numeric_limits<T>::epsilon` return `T`. So for the sake of completeness, i would still use `Number` here. It would cast to `double` in the new function.

Reply:
Alright, this convinces me... In the complex case we would do this in the user code...